lib/geo.js

```diff
@@ -138,11 +138,18 @@ function GeoPoint(data) {
     };
   }
 
-  assert(Array.isArray(data) || typeof data === 'object' || typeof data === 'string', 'must provide valid geo-coordinates array [lat, lng] or object or a "lat, lng" string');
+  assert(
+    Array.isArray(data) ||
+    typeof data === 'object' ||
+    typeof data === 'string',
+    'must provide valid geo-coordinates array [lat, lng] or object or a ' +
+      '"lat, lng" string'
+  );
```

Comment:
We don't have an explicit rule for this, but I think we usually put the closing brackets on the preceding line.